src/libs/ReportUtils.ts

```diff
@@ -8178,6 +8178,33 @@ function isEmptyReport(report: OnyxEntry<Report>, isReportArchived = false): boo
     return generateIsEmptyReport(report, isReportArchived);
 }
 
+/**
+ * Checks if there are any empty (no money) open expense reports for a specific policy and user.
+ * An empty report is defined as having total === 0 and nonReimbursableTotal === 0.
+ * This excludes reports that are being deleted or have errors.
+ */
+function hasEmptyReportsForPolicy(reports: OnyxCollection<Report>, policyID: string | undefined, accountID: number): boolean {
+    if (!policyID || !Number.isFinite(accountID)) {
```

Comment:
I'm not sure if `isFinite` is the correct way. What if `accountID` equals `CONST.DEFAULT_NUMBER_ID`?


```suggestion
    if (!policyID || accountID === CONST.DEFAULT_NUMBER_ID) {
```